Sources/Common/Core/Math/index.js

```diff
@@ -17,7 +17,17 @@ function notImplemented(method) {
   return () => vtkErrorMacro(`vtkMath::${method} - NOT IMPLEMENTED`);
 }
 
-function createArray(size = 3) {
+// Swap columns for n by n matrix
+function swapRowsOrColumnsMatrix_nxn(matrix, n, col1, col2) {
```

Comment:
why name it "swapRows" if it only swap columns ? -> swapColumnsMatrix_nxn

Reply:
Some confusion on my side (due to the fact that both swaping columns and rows are required in different part of the code), last commit should solve this.